test/test.do

```diff
@@ -191,6 +191,17 @@ graph drop g3
 
 graph drop _all
 
+*Interaction Weighted Estimator (Sun and Abraham, 2021) 
+*Generate the variable that indicates cohort
+gen timet=t if z==1
+by i: egen time_of_treat=min(timet)
+*Generate the variable that indicates the control cohort. We use the never treated units as the control cohort. 
+gen never_treat=time_of_treat==.
+*Estimate the event-time coefficients with the IW Estimator.
+xtevent y eta , policyvar(z) window(5) vce(cluster i) impute(nuchange) cohort(time_of_treat) control_cohort(never_treat) 
+*Use reghdfe as the underlying estimation command, this produces identical estimations as with EventStudyInteract.

```

Comment:
@Constantino-Carreto-Romero do not abbreviate as "IW". No need to talk about EventStudyInteract either.

Reply:
I implemented it in https://github.com/JMSLab/xtevent/commit/3f0c7f70a72901a4c458a1e5f43e577ec6eea8b2